### DISCUSSION TITLE
How to integrate with a chatbot

### DISCUSSION THREAD

**Discussion by stevesolun** [niccokunzmann/open-web-calendar#326] [Q&A]
I have a chatbot developed in Python 3. User comes from FB and sends a message, Python catches it and transfers to Dialogflow NLU. Dialogflow returns the intent to Python and business logic applied on it.

I want to add reminders functionality to my bot - a date and time picker where my user will choose date and time to remind him to do X.

I will pick it from the webview and write to the DB.

Please advise how can I set dates and times and integrate it?

---
<small>
We're using [Polar.sh](https://polar.sh/niccokunzmann) so you can upvote and help fund this issue. We receive the funding once the issue is completed & confirmed by you. Thank you in advance for helping prioritize & fund our work.
</small>

<a href="https://polar.sh/niccokunzmann/open-web-calendar/issues/70">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/70/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/niccokunzmann/open-web-calendar/issues/70/pledge.svg">
</picture>
</a>

**Comment by niccokunzmann** [comment]
Hello @stevesolun, this calendar consists of two parts:
1. The Frontend which is "DHTMLX scheduler" but could also be "fullcalendar" #38 
2. The backend which takes ICS files and prepares them for the frontend.

I think that you may use one of the front-ends independently of this software for your use case to display events picked by the user.
If you like to use this software here, we can work on #71 or integrating modules of dhtmlx which allow users to pick and delete dates. See the dhtmlx scheduler documentation for this. Your bot would then output ICS dates.

**Comment by stevesolun** [comment]
@niccokunzmann can we make a zoom meeting today to discuss it?
In general I want to use Facebook messenger webview, my user will choose times and dates from a given calendar as reminders to do something, I need to know how to get them and write to my DB.

Can you help me with this case?

**Comment by niccokunzmann** [comment]
Hello,

I do not want to work as consultant right now. In generally, I think I
am not the right person for this task - others have worked more with the
chatbot or the calendar frontends. Please ask in their communities.
I am occupied with other deeds.

I am sure you will find next steps and help from someone else in the
mean time.
